gnd/src/main/java/com/google/android/gnd/ui/basemapselector/BasemapSelectorFragment.java

```diff
@@ -0,0 +1,74 @@
+package com.google.android.gnd.ui.basemapselector;
+
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.core.view.ViewCompat;
+import androidx.core.view.WindowInsetsCompat;
+
+import com.google.android.gnd.MainViewModel;
+import com.google.android.gnd.R;
+import com.google.android.gnd.databinding.BasemapSelectorFragBinding;
+import com.google.android.gnd.inject.ActivityScoped;
+import com.google.android.gnd.ui.common.AbstractFragment;
+import com.google.android.gnd.ui.map.MapProvider;
+import com.google.android.gnd.ui.map.MapProvider.MapAdapter;
+
+import javax.inject.Inject;
+
+import io.reactivex.Single;
+
+@ActivityScoped
```

Comment:
Please add short description of what this fragment represents.

Reply:
Done

Not thrilled with the text, but it gets the point across.